rllib/tests/test_io.py

```diff
@@ -151,25 +150,25 @@ def test_agent_input_postprocessing_enabled(self):
             self.assertEqual(result["timesteps_total"], 250)  # read from input
             self.assertTrue(np.isnan(result["episode_reward_mean"]))
 
-    def test_agent_input_eval_sim(self):
-        for fw in framework_iterator():
+    def test_agent_input_eval_sampler(self):
+        for fw in ["torch"]:
```

Comment:
Please use
```
config = ...

for fw in framework_iterator(config, frameworks="torch"):
```
instead for consistency.

Reply:
That was supposed to have TF too, fixed now